Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Templates/FieldsAndConstructor.tt

```diff
@@ -8,18 +8,32 @@
     if (_model.LambdaMethod.UsingDependencyInjection)
     {
 #>
-        private readonly ServiceProvider serviceProvider;
+        private <#= _model.IsExecutable ? "static " : "" #>readonly ServiceProvider serviceProvider;
 <#
     }
     else
     {
 #>
-        private readonly <#= _model.LambdaMethod.ContainingType.Name #> <#= _model.LambdaMethod.ContainingType.Name.ToCamelCase() #>;
+        private <#= _model.IsExecutable ? "static " : "" #>readonly <#= _model.LambdaMethod.ContainingType.Name #> <#= _model.LambdaMethod.ContainingType.Name.ToCamelCase() #>;
+        private <#= _model.IsExecutable ? "static " : "" #>readonly <#= _model.Serializer #> serializer;
 <#
     }
 #>
 
+<#
+    if (_model.IsExecutable)
```

Comment:
Remove the static condition and static constructor. We are always using the instance constructor.